ios/BT/bridge/ENPermissionsModule.m

```diff
@@ -10,11 +11,19 @@ @implementation ENPermissionsModule
 RCT_EXPORT_MODULE();
 
 RCT_EXPORT_METHOD(requestExposureNotificationAuthorization: (RCTResponseSenderBlock)callback) {
-  [[ExposureManager shared] requestExposureNotificationAuthorizationWithEnabled:YES callback:callback];
+  [[ExposureManager shared] requestExposureNotificationAuthorizationWithEnabled:YES callback:^(NSError * _Nullable error) {
```

Comment:
It would be great if we could refactor these to use `Promise`s instead of callbacks (would require changing the corresponding function signatures in  `nativeModule.ts`)

Reply:
that will also make an impact on the Android side? This change is so to divide the native component from the RN side if we have RN callbacks I think we are mixing the layers

Reply:
good question - yes, it would require changes on the android side. My thinking is that it would be good for the native layer to have a consistent API (`nativeModule` currently has a mixture of promises and callbacks), but I would say that it's low priority, and as you point out it would be a little disruptive because it would require coordination with thee Android side

Reply:
I'll speak with Juan about this, maybe we can coordinate the effort